src/osd/OSD.cc

```diff
@@ -10220,7 +10227,7 @@ void OSD::maybe_override_cost_for_qos()
   // If the scheduler enabled is mclock, override the default PG deletion cost
   // so that mclock can meet the QoS goals.
   if (op_queue_type_t::mClockScheduler == osd_op_queue_type()) {
-    uint64_t pg_delete_cost = 15728640;
+    uint64_t pg_delete_cost = 1048576;
```

Comment:
How would a user adjust this value if this didn't turn out to be appropriate in some specific deployment?